ga4gh/backend.py

```diff
@@ -164,13 +164,19 @@ def _search(self, start, end):
 
     @classmethod
     def _getStart(cls, readAlignment):
-        return readAlignment.alignment.position.position
+        if readAlignment.alignment is None:
+            return -2
```

Comment:
Changes in this file are hacks for now.  The reason we need this is because `getReadAlignments` is returning unmapped reads and we can't use the interval iterator algorithm on anything that doesn't have `alignment` objects.

Should we filter out all unmapped reads within `getReadAlignments` or, alternatively, pass a parameter to `getReadAlignments` to indicate that we only want mapped reads?  Note that in this case it is returning unmapped reads even though we're passing in a non-null reference.